vita3k/vkutil/include/vkutil/vkutil.h

```diff
@@ -24,8 +24,18 @@
 #include <vulkan/vulkan.hpp>
 #define VMA_STATIC_VULKAN_FUNCTIONS 0
 #define VMA_DYNAMIC_VULKAN_FUNCTIONS 1
+
+#if (defined(_WIN64) && defined(__clang__))
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnullability-completeness"
+#endif
+
 #include <vk_mem_alloc.hpp>
 
+#if (defined(_WIN64) && defined(__clang__))
+#pragma clang diagnostic pop
+#endif
```

Comment:
Is the `defined(_WIN64)` necessary?

Reply:
This check was used by default for macos and ios (now it's on for all targets), So I think switching it on on windows is a bug in the library, and for macos it's desired behavior. No idea about linux. so maybe `!__APPLE__` is better

Reply:
The nullability warnings also appear on Linux (you can look at the log of the github ubuntu action), so I think if it does not cause any issue, just putting `#ifdef __clang__` would be fine (that's not an issue for it to be already defined on MacOS)

Reply:
you are right. done.